### PR TITLE
fix(cpplint): include what you use - perception

### PR DESCRIPTION
**Pull request by xmfcx**
## Description

**Parent Issue:**
- https://github.com/autowarefoundation/autoware.universe/issues/9556

Extracted from:
- https://github.com/autowarefoundation/autoware.universe/pull/9558

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: perception/autoware_bytetrack/lib/include/lapjv.h [hunk 1]
```diff
@@ -66,6 +66,8 @@
 
 #if 0
 #include <assert.h>
+
+#include <cstdio>
```
Comment (mitsudome-r) [inline perception/autoware_bytetrack/lib/include/lapjv.h:70]: I think we can remove this change because we will be covering it by another PR: https://github.com/autowarefoundation/autoware.universe/pull/9563/files

Comment (mitsudome-r) [reply perception/autoware_bytetrack/lib/include/lapjv.h:70]: I'm still going to approve it because there will be no harm. (You will just have a conflict in the above PR)

**Review by mitsudome-r — APPROVED** [review]

File: perception/autoware_bytetrack/lib/src/byte_tracker.cpp
Comment (mitsudome-r) [inline perception/autoware_bytetrack/lib/src/byte_tracker.cpp:1]: @xmfcx Do you have any idea why GitHub is failing to show the correct diffs in this file?

File: perception/autoware_bytetrack/lib/include/lapjv.h (hunk 1 above)
Comment (xmfcx) [reply perception/autoware_bytetrack/lib/include/lapjv.h:70]: Good catch!

I'll remove it now.

File: perception/autoware_bytetrack/lib/src/byte_tracker.cpp
Comment (xmfcx) [reply perception/autoware_bytetrack/lib/src/byte_tracker.cpp:1]: image

It had bad line endings it seems:

**Before (main):**
```console
mfc@whale:~$ file -k /home/mfc/projects/autoware/src/universe/autoware.universe/perception/autoware_bytetrack/lib/src/byte_tracker.cpp
/home/mfc/projects/autoware/src/universe/autoware.universe/perception/autoware_bytetrack/lib/src/byte_tracker.cpp: C source, ASCII text, with CRLF line terminators
```

**After (This PR):**
```console
mfc@whale:~$ file -k /home/mfc/projects/autoware/src/universe/autoware.universe/perception/autoware_bytetrack/lib/src/byte_tracker.cpp
/home/mfc/projects/autoware/src/universe/autoware.universe/perception/autoware_bytetrack/lib/src/byte_tracker.cpp: C source, ASCII text
```

File: perception/autoware_bytetrack/lib/include/lapjv.h (hunk 1 above)
Comment (xmfcx) [reply perception/autoware_bytetrack/lib/include/lapjv.h]: Reverted with https://github.com/autowarefoundation/autoware.universe/pull/9569/commits/2b170200c51c60969f530dd545174c4709a64c98
